.github/workflows/build-main.yml

```diff
@@ -39,9 +39,6 @@ jobs:
       with:
         image: ${{ env.IMAGE_REGISTRY }}/operator-certification-operator-bundle
         tags: ${{ github.sha }} latest
```

Comment:
Do we even care about the sha build now?

Reply:
It's there mostly for tracking daily development, and allowing someone to create an index pointing specifically to a bundle they'd like to test.
Not sure if that's a common use case though.

Reply:
@acornett21 @yashoza19 @spabba17 is there a use case where Preflight is run on an index that points to a specific bundle (not "latest")?
Or a catalog source created for such an index?

Reply:
If someone wants to do that, they can push to their own quay.io repo. I don't see the utility of it honestly.

Reply:
in that case I'll get rid of it,  if no objections.